packages/matchbox/src/components/Popover/styles.js

```diff
@@ -0,0 +1,46 @@
+import { tokens } from '@sparkpost/design-tokens';
+
+export const content = props => `
+  position: absolute;
+  background: ${tokens.color_white};
+  border: ${tokens.borderWidth_100} solid ${tokens.color_gray_400};
+  border-radius: ${tokens.borderRadius_100};
+  box-shadow: ${tokens.boxShadow_100};
+  margin-top: ${props.isTop ? tokens.spacing_0 : tokens.spacing_100};
+  margin-bottom: ${props.isTop ? tokens.spacing_100 : tokens.spacing_0};
+
+  left: ${props.isLeft ? 'auto' : '0'};
+  right: ${!props.isLeft ? 'auto' : '0'};
+  top: ${props.isTop ? 'auto' : '100%'};
+  bottom: ${!props.isTop ? 'auto' : '100%'};  
+`;
+
+export const transition = props => {
+  switch (props.state) {
+    case 'entering':
+    case 'entered':
+      return `
+        pointer-events: auto;
+        visibility: visible;
+        opacity: 1;
+        transform: translateY(0);
+        transition: opacity ${tokens.motionDuration_medium} ${tokens.motionEase_out},
```

Comment:
I think the transition state falls out of sync with the component state which causes the popover to get stuck if you click the activator a couple times.

<img width="1008" alt="Screen Shot 2020-04-06 at 1 37 23 PM" src="https://user-images.githubusercontent.com/1335605/78588062-f2b2d100-780b-11ea-9005-9a4727007ff5.png">

Reply:
I don't think I can reproduce this, but I can try to disable pointer events until content has been fully rendered?

Reply:
I think this might be a bug with storybook actions – can't tell if you have them enabled. Sometimes they freeze the browser